documentation/src/main/asciidoc/ch01.asciidoc

```diff
@@ -86,69 +86,6 @@ Note that adding this dependency is usually not required for applications runnin
 application server. You can learn more about the integration of Jakarta Bean Validation and CDI in
 <<section-integration-with-cdi>>.
 
-==== Updating Hibernate Validator in WildFly
```

Comment:
Maybe keep the section, but mention it's no longer possible as of Hibernate Validator 9.0?

By the way, is there a migration guide you should be updating?

Reply:
Hmm, ok 😃 will do.
I found a migration guide here https://github.com/hibernate/hibernate.org/blob/production/validator/documentation/migration-guide.adoc
I guess ... I'll open a draft PR, and we can keep adding things as we go and merge it on the release day.